<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemMark</class>
 <widget class="QWidget" name="itemMark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加账目</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/img_asset/icon_asset/2x_save-page.png</normalon>
   </iconset>
  </property>
  <widget class="QLineEdit" name="moneyLine">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>30</y>
     <width>316</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>30</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 220, 226);</string>
   </property>
   <property name="text">
    <string>金额</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>80</y>
     <width>316</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777211</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>饮食</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>娱乐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教育</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>80</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 220, 226);</string>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemMarkBackGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img_asset/icon_asset/TIM截图20190706102442.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="content">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>121</y>
     <width>316</width>
     <height>109</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>160</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 220, 226);</string>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <zorder>itemMarkBackGround</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_3</zorder>
  <zorder>moneyLine</zorder>
  <zorder>label</zorder>
  <zorder>content</zorder>
  <zorder>label_2</zorder>
  <zorder>cancel_btn</zorder>
  <zorder>confirm_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
